internal/controller/resourcesetinputprovider_controller_git_test.go

```diff
@@ -0,0 +1,906 @@
+// Copyright 2025 Stefan Prodan.
+// SPDX-License-Identifier: AGPL-3.0
+
+package controller
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/fluxcd/pkg/apis/meta"
+	"github.com/fluxcd/pkg/cache"
+	"github.com/fluxcd/pkg/git/github"
+	"github.com/fluxcd/pkg/runtime/conditions"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/yaml"
+
+	fluxcdv1 "github.com/controlplaneio-fluxcd/flux-operator/api/v1"
+	"github.com/controlplaneio-fluxcd/flux-operator/internal/testutils"
+)
+
+func TestResourceSetInputProviderReconciler_GitTag_LifeCycle(t *testing.T) {
```

Comment:
```suggestion
func TestResourceSetInputProviderReconciler_GitLabTag_LifeCycle(t *testing.T) {
```

Or rename the next two tests below to `...GitBranch...` and `...GitChangeRequest...`

Reply:
Oh ok, I missed it! Thanks